src/components/Button.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export default React.forwardRef<
+  HTMLButtonElement,
+  React.HTMLProps<HTMLButtonElement>
+>(function Button({ onClick, children }, reference) {
```

Comment:
**Nitpick**

Should we past `...rest` props on to the underlying button element?